esp-hal-common/src/rtc_cntl/rtc/esp32_sleep.rs

```diff
@@ -223,61 +225,97 @@ impl RtcSleepConfig {
         // settings derived from esp-idf after basic boot
         unsafe {
             let rtc_cntl = &*esp32::RTC_CNTL::ptr();
-            #[rustfmt::skip]
-            rtc_cntl.options0.modify(|_, w| w
-                .bias_core_force_pu().clear_bit()
-                .bias_core_folw_8m().set_bit()
-                .bias_i2c_force_pu().clear_bit()
-                .bias_i2c_folw_8m().set_bit()
-                .bias_force_nosleep().clear_bit()
-                .bias_sleep_folw_8m().set_bit()
-                .xtl_force_pu().clear_bit()
-            );
-            #[rustfmt::skip]
-            rtc_cntl.reg.modify(|_, w| w
-                .force_pu().clear_bit()
-                .dboost_force_pu().clear_bit()
-                .dboost_force_pd().set_bit()
-            );
-            #[rustfmt::skip]
-            rtc_cntl.pwc.modify(|_, w| w
-                .slowmem_force_pu().clear_bit()
-                .fastmem_force_pu().clear_bit()
-                .force_noiso().clear_bit()
-                .slowmem_force_noiso().clear_bit()
-                .fastmem_force_noiso().clear_bit()
-            );
-            #[rustfmt::skip]
-            rtc_cntl.dig_pwc.modify(|_, w| w
-                .dg_wrap_force_pu().clear_bit()
-                .wifi_force_pu().clear_bit()
-                .wifi_force_pd().set_bit()
-                .inter_ram4_force_pu().clear_bit()
-                .inter_ram3_force_pu().clear_bit()
-                .inter_ram2_force_pu().clear_bit()
-                .inter_ram1_force_pu().clear_bit()
-                .inter_ram0_force_pu().clear_bit()
-                .rom0_force_pu().clear_bit()
-                .lslp_mem_force_pu().clear_bit()
-            );
-            #[rustfmt::skip]
-            rtc_cntl.dig_iso.modify(|_, w| w
-                .dg_wrap_force_noiso().clear_bit()
-                .wifi_force_noiso().clear_bit()
-                .wifi_force_iso().set_bit()
-                .inter_ram4_force_noiso().clear_bit()
-                .inter_ram3_force_noiso().clear_bit()
-                .inter_ram2_force_noiso().clear_bit()
-                .inter_ram1_force_noiso().clear_bit()
-                .inter_ram0_force_noiso().clear_bit()
-                .rom0_force_noiso().clear_bit()
-                .dg_pad_force_unhold().clear_bit()
-                .dg_pad_force_noiso().clear_bit()
-            );
-            #[rustfmt::skip]
-            rtc_cntl.int_ena.modify(|_, w| w
-                .brown_out_int_ena().set_bit()
-            );
+
+            rtc_cntl.options0.modify(|_, w| {
+                w.bias_core_force_pu()
```

Comment:
To be honest I find the old version more readable here. It's much easier to see at a glance which bits are manipulated IMO.